verification/curator-service/ui/src/components/App/App.tsx

```diff
@@ -774,14 +774,17 @@ export default function App(): JSX.Element {
                             <PolicyLink
                                 type="cookie-policy"
                                 classes={{
-                                    root: clsx([
-                                        classes.link,
-                                        classes.lastLink,
-                                    ]),
+                                    root: clsx([classes.link]),
                                 }}
                             >
                                 Cookie policy
                             </PolicyLink>
+                            <a
+                                href="mailto:info@global.health?subject=Feedback regarding Global.health map"
+                                className={`${classes.link} ${classes.lastLink}`}
```

Comment:
It is a good practice to use library like `clsx` in order to use two class names. This library is already installed and used in this project.